Base/etc/ca_certs.ini

```diff
@@ -0,0 +1,465 @@
+country=US
+
+[GTS Root R3]
+country=US
+
+[GTS Root R4]
+country=US
+
+[GeoTrust Global CA]
+country=US
+
+[GeoTrust Primary Certification Authority]
+country=US
+
+[(c) 2007 GeoTrust Inc. - For authorized use only]
+country=US
+
+[(c) 2008 GeoTrust Inc. - For authorized use only]
+country=US
+
+[GeoTrust Universal CA]
+country=US
+
+[GeoTrust Universal CA 2]
+country=US
+
+[GlobalSign ECC Root CA - R4]
+country=
+
+[GlobalSign ECC Root CA - R5]
+country=
+
+[Root CA]
+country=BE
+
+[GlobalSign Root CA - R2]
+country=
+
+[GlobalSign Root CA - R3]
+country=
+
+[GlobalSign Root CA - R6]
+country=
+
+[Global Chambersign Root - 2008]
+country=EU
+
+[Go Daddy Class 2 Certification Authority]
+country=US
+
+[Go Daddy Root Certificate Authority - G2]
+country=US
+
+[Hellenic Academic and Research Institutions ECC RootCA 2015]
+country=GR
+
+[Hellenic Academic and Research Institutions RootCA 2011]
+country=GR
+
+[Hellenic Academic and Research Institutions RootCA 2015]
+country=GR
+
+[Hongkong Post Root CA 1]
+country=HK
+
+[Hongkong Post Root CA 3]
+country=HK
+
+[ISRG Root X1]
+country=US
+
+[IdenTrust Commercial Root CA 1]
+country=US
+
+[IdenTrust Public Sector Root CA 1]
+country=US
+
+[Izenpe.com]
+country=ES
+
+[LuxTrust Global Root 2]
+country=LU
+
+[Microsec e-Szigno Root CA 2009]
+country=HU
+
+[Tan\C3\BAs\C3\ADtv\C3\A1nykiad\C3\B3k (Certification Services)]
+country=HU
+
+[Network Solutions Certificate Authority]
+country=US
+
+[Copyright (c) 2005]
+country=CH
+
+[OISTE Foundation Endorsed]
+country=CH
+
+[OISTE Foundation Endorsed]
+country=CH
+
+[Root Certification Authority]
+country=BM
+
+[QuoVadis Root CA 1 G3]
+country=BM
+
+[QuoVadis Root CA 2]
+country=BM
+
+[QuoVadis Root CA 2 G3]
+country=BM
+
+[QuoVadis Root CA 3]
+country=BM
+
+[QuoVadis Root CA 3 G3]
+country=BM
+
+[SSL.com EV Root Certification Authority ECC]
+country=US
+
+[SSL.com EV Root Certification Authority RSA R2]
+country=US
+
+[SSL.com Root Certification Authority ECC]
+country=US
+
+[SSL.com Root Certification Authority RSA]
+country=US
+
+[SZAFIR ROOT CA2]
+country=PL
+
+[SecureSign RootCA11]
+country=JP
+
+[SecureTrust CA]
+country=US
+
+[Secure Global CA]
+country=US
+
+[Security Communication RootCA2]
+country=JP
+
+[Security Communication RootCA1]
+country=JP
+
+[Sonera Class2 CA]
+country=FI
+
+[Staat der Nederlanden EV Root CA]
+country=NL
+
+[Staat der Nederlanden Root CA - G2]
+country=NL
+
+[Staat der Nederlanden Root CA - G3]
+country=NL
+
+[Starfield Class 2 Certification Authority]
+country=US
+
+[Starfield Root Certificate Authority - G2]
+country=US
+
+[Starfield Services Root Certificate Authority - G2]
+country=US
+
+[SwissSign Gold CA - G2]
+country=CH
+
+[SwissSign Platinum CA - G2]
+country=CH
+
+[SwissSign Silver CA - G2]
+country=CH
+
+[Digital Certificate Services]
+country=ch
+
+[Symantec Trust Network]
+country=US
+
+[Symantec Trust Network]
+country=US
+
+[Symantec Trust Network]
+country=US
+
+[Symantec Trust Network]
+country=US
```

Comment:
It'd be nice to not trust Symantec. They are a shady CA.
https://security.googleblog.com/2017/09/chromes-plan-to-distrust-symantec.html

(I know this doesn't matter right now because the signatures aren't verified, but it'll be relevant when Serenity starts doing that.)

Reply:
Ideally, we'd run our own trust program!
I think, for the time being, we can afford to let this be, and when we implement a blacklist for these, add Symantec to that.